src/flow-control/log-timings.js

```diff
@@ -0,0 +1,57 @@
+const Rx = require('rxjs');
+const _ = require('lodash');
+const { bufferCount, take } = require('rxjs/operators');
+const BaseHandler = require('./base-handler');
+
+module.exports = class LogTimings extends BaseHandler {
+    printExitInfoTimingTable(exitInfos) {
+        const exitInfoTable = _(exitInfos)
+            .map(({ command, timings, index, killed, exitCode }) => {
+                const readableDurationMs = (timings.endDate - timings.startDate).toLocaleString();
+                return {
+                    'call-index': index,
```

Comment:
Is this meant to be different from `(index)`?

```
$ concurrently --show-timings "echo hi"
[0] echo hi started at 11/18/2021, 10:12:25 PM
[0] hi
[0] echo hi stopped at 11/18/2021, 10:12:25 PM after 18ms
[0] echo hi exited with code 0

Timings:
┌─────────┬────────────┬──────┬──────────┬───────────┬────────┬───────────┐
│ (index) │ call-index │ name │ duration │ exit-code │ killed │  command  │
├─────────┼────────────┼──────┼──────────┼───────────┼────────┼───────────┤
│    0    │     0      │  ''  │ '~18ms'  │     0     │ false  │ 'echo hi' │
└─────────┴────────────┴──────┴──────────┴───────────┴────────┴───────────┘
```

Reply:
so the `(index)` is the index in the sorted array, `call-index` is the index in terms of the order the processes were started, do you have an idea for a better name to make this clear? the `(index)` column is auto generated so cant change that easily unfortunately

Reply:
maybe it doesnt matter here actually, I'll remove the call index and just leave the sorted index